build_web_compilers/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 3.1.0-dev
+
+- Add the `emit-debug-symbols` option for the `build_web_compilers:ddc`
+  builder. The debug symbols output is used by the debugger in webdev for
+  variable inspection. This setting is disabled by default but can be
```

Comment:
Should we consider enabling this by default, or even making it non-configurable?

How many users do we expect to want this?

Reply:
Eventually everyone will use it, but the feature will not be ready for some time. I am hiding it now since it is still in development stage, and plan to enable it after dogfooding.